rust/src/lib/nm/connection.rs

```diff
@@ -67,6 +68,32 @@ pub(crate) fn perpare_nm_conns(
             conn_matcher,
             gen_conf_mode,
         )? {
+            // Clear forwarding when not supported by NetworkManager to prevent
+            // failures. TODO: Remove this code once the minimum
+            // supported NetworkManager version is >= 1.54
+            if !forwarding_supported {
+                if let Some(ipv4) = nm_conn.ipv4.as_mut() {
+                    ipv4.forwarding = None;
+                }
+
+                if let Some(desired_iface) = merged_iface.desired.as_ref() {
+                    let forwarding_desired = desired_iface
+                        .base_iface()
+                        .ipv4
+                        .as_ref()
+                        .and_then(|ip4| ip4.forwarding)
+                        .is_some();
+
+                    if forwarding_desired {
+                        log::warn!(
+                            "Clearing unsupported ipv4.forwarding for \
```

Comment:
This sounds like nmstate does not support ipv4.forwarding, could you rephrase it indicate NetworkManager too old?

Reply:
I rephrased it, thanks !